src/main/java/com/filip/workshop_managment/service/VehicleServiceImpl.java

```diff
@@ -18,21 +18,21 @@ public VehicleServiceImpl(VehicleRepository vehicleRepository) {
     }
 
     @Override
-    public List<Vehicles> listAll() {
-        List<Vehicles> vehicles = new ArrayList<>();
+    public List<Vehicle> listAll() {
+        List<Vehicle> vehicles = new ArrayList<>();
        vehicleRepository.findAll().forEach(vehicles::add);
         return vehicles;
     }
 
     @Override
-    public Vehicles getById(Long id) {
+    public Vehicle getById(Long id) {
```

Comment:
Unikaj nulli jak ognia.
Polecam Optionale. W tym przypadku:
```suggestion
    public Optional<Vehicle> getById(Long id) {
```